spec/workers/work_packages/apply_working_days_change_job_spec.rb

```diff
@@ -1811,26 +1811,4 @@
       end
     end
   end
-
-  describe '.scheduled?' do
```

Comment:
Related to [this](https://github.com/opf/openproject/pull/12998/files#diff-254057ccc73a0d0229fc0f8cb7c67b1f26c2539c43e5ad90a59f3eec0e5dc7f3R46) perhaps it makes sense to still have a (although universal) `.scheduled?` kind of method in application job and use that in the linked spot and test it in a separate spec.